templates/.github/workflows/test.yml

```diff
@@ -16,6 +16,16 @@ jobs:
       - run: npm install
       - run: npm run test:schema
 
+  lint:
```

Comment:
We may create a separate lint workflow, what do you think?

Reply:
I think as it does not fix the linting, it can be considered as a test. Therefore, I'd keep it there